index.html

```diff
@@ -132,5 +132,29 @@ <h3>LETS CONNECT</h3>
         </ul>
     </div>
 </section>
+<section class="contact-form">
+    <h1>Contact me</h1>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form method="" action="">
+        <ul>
+            <li>
+                <label for="name">
+                <input type="text" id="name" name="user-name" placeholder="Full Name" maxlength="30" required>
+                </label>
+            </li>
+            <li>
+                <label for="mail">
+                <input type="email" id="email" name="user-email" placeholder="Email" maxlength="30" required>
+                </label>
+            </li>
+            <li>
+                <label for="msg">
+                <textarea id="message" name="user_message" maxlength="500">Write your message here...</textarea>
+                </label>
+            </li>
+        </ul>
+        <button type="submit">Get in touch</button>
```

Comment:
good job on implementing this part , Only one small issue to mention here your button has a different padding and font size , please make sure to replicate figma design as you can.
**your button:**

![button padding student temp1](https://user-images.githubusercontent.com/59707859/220474273-1cbdd2b4-c08a-4f90-8d3e-6d6a912e6eab.PNG)

**figma design:**

![button padding temp1](https://user-images.githubusercontent.com/59707859/220474278-2ada2e82-5daa-4f89-8301-d6c1e770f8eb.PNG)